### PR TITLE
Use Travis to ensure that provided examples pass all tests

### PR DESCRIPTION
**Pull request by ryanplusplus**
This assumes:
- all examples are contained in a single file
- all source files are loaded from the tests with <exercise name>.lua
- all examples are valid in Lua 5.3

### CODE REVIEW

File: exercises/space-age/example.lua [hunk 1]
```diff
@@ -22,7 +22,7 @@ function SpaceAge:new(seconds)
   local meta_tbl = { seconds = seconds }
   for k, v in pairs(SpaceAge.ORBITAL_PERIODS) do
     local val = round(seconds / SpaceAge.EARTH_ORBITAL_PERIOD / v, 2)
-    meta_tbl['on_' .. k] = loadstring('return ' .. val)
+    meta_tbl['on_' .. k] = load('return ' .. val)
```
Comment (ryanplusplus) [inline exercises/space-age/example.lua]: Updated to be valid Lua 5.3 without any compatibility patches

File: exercises/alphametics/example.lua [hunk 2]
```diff
@@ -59,7 +59,7 @@ local function populate_letter_map(letter_map, letters, values)
 end
 
 local function evaluates_true(solution)
-  return loadstring('return ' .. solution)()
+  return load('return ' .. solution)()
```
Comment (ryanplusplus) [inline exercises/alphametics/example.lua]: Updated to be valid Lua 5.3 without any compatibility patches

**Comment by aarti** [comment]
Nice :+1:

**Comment by ryanplusplus** [comment]
Thanks @aarti :)

**Pull request merged**